imports/plugins/core/dashboard/client/components/toolbar.js

```diff
@@ -80,7 +78,7 @@ class PublishControls extends Component {
       <FlatButton
         onClick={() => {
           Reaction.showActionView({
-            i18nKeyTitle: "dashboard.coreTitle",
+            i18nKeyTite: "dashboard.coreTitle",
```

Comment:
I don't see this being fixed?
https://github.com/reactioncommerce/reaction/blob/dc2703a93df5d231e432f014de965e9518876467/imports/plugins/core/dashboard/client/components/toolbar.js#L81-L81